src/frontend/src/test-e2e/util.ts

```diff
@@ -1,17 +1,23 @@
 import { remote } from "webdriverio";
 import { command } from "webdriver";
 import { WebAuthnCredential } from "../../test-setup";
+import { ChromeOptions } from "@wdio/types/build/Capabilities";
+import * as fsasync from "fs/promises";
+import * as fs from "fs";
 
 // mobile resolution is used when env variable SCREEN=mobile is set
 const MOBILE_SCREEN: ScreenConfiguration = {
   screenType: "mobile",
-  windowSize: "360,640",
+  deviceMetrics: {
+    width: 360,
+    height: 640,
+  },
 };
 
 // desktop resolution is used when env variable SCREEN=desktop is set
 const DESKTOP_SCREEN: ScreenConfiguration = {
   screenType: "desktop",
-  windowSize: "1920,1080",
+  windowSize: "1928,1209", // increased from 1920,1080 when the headless flag was removed to account for the GUI border of 8,129.
```

Comment:
future contributors won't care that there was a headless flag; also what about the GUI border? why trim it?

Reply:
Agreed, the idea initially was to keep the diff smaller. But apparently there are minor changes in font rendering when not using headless mode. So this point is moot. I Will create a PR reverting this to 1920*1080.

Reply:
#750